src/api/common/schemas/get-ledger.json

```diff
@@ -14,7 +14,12 @@
     "parentLedgerHash": {"$ref": "hash256"},
     "parentCloseTime": {"type": "integer", "minimum": 0},
     "totalDrops": {"$ref": "value"},
-    "transactionHash": {"$ref": "hash256"}
+    "transactionHash": {"$ref": "hash256"},
+    "transactions": {"type": "array", "items": {"type": "object"}},
+    "rawTransactions": {"type": "string"},
+    "transactionHashes": {"type": "array", "items": {"$ref": "hash256"}},
+    "rawAccounts": {"type": "string"},
```

Comment:
Yeah, but there are some things in the rippled format that we don't want to expose because they are meaningless to users like `BookNode` and `OwnerNode`, but I'm not sure if they factor into the hash. So it may be possible to eliminate the `raw` fields; it would just require some research to see.